flink-dist/pom.xml

```diff
@@ -679,7 +679,7 @@ under the License.
 						<goals>
 							<goal>shade</goal>
 						</goals>
-						<configuration combine.self="override">
```

Comment:
this removal is safe because it is a distinct execution, and hence never had any effect.